doc/_templates/support-button.html

```diff
@@ -0,0 +1 @@
+<a class="btn btn-secondary support-button" href="https://mybinder.readthedocs.io/en/latest/about/support.html" role="button"><i class="fas fa-heart"></i>Support Binder</a>
```

Comment:
This unfortunately leads to a self-referencing page link on the support page, but maybe we just live with it for now?

Reply:
IMO it's OK to have this for now - we can try to improve the UX in the future if that's OK w/ you